app/templates/public/index.hbs

```diff
@@ -1,3 +1,6 @@
 <div class="event description">
   {{sanitize model.event.description}}
-</div>
\ No newline at end of file
+  <div class="row">
+    {{speaker-list speakers = speakers}}
```

Comment:
No space around `=`. Not sure why the template lint did not fail for this. Will check that later though.